e2e/test/scenarios/admin-2/tools/erroring-questions.cy.spec.js

```diff
@@ -125,7 +125,7 @@ describe("admin > tools > erroring questions ", { tags: "@quarantine" }, () => {
 
   describe("when feature disabled", () => {
     beforeEach(() => {
-      cy.onlyOn(isEE);
+      onlyEE();
```

Comment:
I think this pattern is more ergonomic, so I replaced it everywhere.

Reply:
`onlyOn(foo)` is more semantic, imo.

`onlyOSS` or `onlyEE` don't mean much to me, but I guess people will simply get used to the pattern. Maybe this is just me, FWIW.

Reply:
I was trying to reduce the number of things someone has to know to get this done, instead of needing to know that there's a (1) function on the `cy` class that can skip tests and (2) there's an `isEE` variable somewhere, you just have to know (1) there's a helper function to restrict this to EE.

lemme think about how to make that one api easier

Reply:
The approach is good.
It's only the name that might be confusing or not clear enough.